plotting/map.py

```diff
@@ -224,31 +221,28 @@ def load_data(self):
                 depth_value_map = depth_value
 
             data = []
-            allvars = []
-            for v in self.variables:
```

Comment:
dead for-loop since we don't do compound variables anymore (i.e. `vozocrtx,vomecrty`).